### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 03cranec**

### CODE REVIEW

**Review by okane16 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,30 +1,17 @@
...
-
-# MooseJS
+<a href="https://www.getmoose.dev/"><img src="https://raw.githubusercontent.com/514-labs/moose/main/logo-m-light.png" alt="moose logo" height="100px"></a>
 
 [![NPM Version](https://img.shields.io/npm/v/%40514labs%2Fmoose-cli?logo=npm)](https://www.npmjs.com/package/@514labs/moose-cli?activeTab=readme)
 [![Moose Community](https://img.shields.io/badge/slack-moose_community-purple.svg?logo=slack)](https://join.slack.com/t/moose-community/shared_invite/zt-2fjh5n3wz-cnOmM9Xe9DYAgQrNu8xKxg)
 [![Docs](https://img.shields.io/badge/quick_start-docs-blue.svg)](https://docs.moosejs.com/)
 [![MIT license](https://img.shields.io/badge/license-MIT-yellow.svg)](LICENSE)
 
-[MooseJS](https://www.moosejs.com/) is an open-source developer framework for your data & analytics stack. Moose is intended for any application where collecting data, processing data, and extracting insights from that data is at the heart of the application. Moose tries to take the decades-old best practices of frontend and backend developer frameworks, and bring them to the world of developing data-driven applications.
```
Comment (oatsandsugar) [inline README.md:10]: or feature?

File: README.md [hunk 2]
```diff
@@ -1,30 +1,17 @@
...
-
-**Highlights**
```
Comment (oatsandsugar) [inline README.md:12]: Can probably get rid of this title and just roll these bullets after the first para

File: README.md [hunk 3]
```diff
@@ -1,30 +1,17 @@
...
-
-- Abstract away middleware and infrastructure complexity: expertise in the data engineering ecosystem not required. MooseJS automatically configures your data stack based on the business logic of your code.
-- Developer-first: Develop in languages you know, in the IDE of your choice. Git-based based workflows for version control and change management. Plug into CI/CD and automated testing workflows. Iterate quickly with local development, then scale to the cloud
-- Hot-reload local dev server: run your whole data/analytics stack locally, and see the impact of changes in real time as you edit code - just like developing a web application
-- Auto-managed migrations: Moose automatically manages schema evolution across your data stack with "Data Change Management", so you don’t break upstream and downstream dependencies
-- End-to-end typed: Moose types all your components from data capture through consumption, so you catch issues at build time, instead of in production
-- Data product APIs: Moose automatically creates API endpoints for your data products, and generates SDKs in the languages of your choice for easy discovery and consumption
-
```
Comment (oatsandsugar) [inline README.md:20]: Anything on like "and its built on the best stuff there is, if you had a $1m data team, they'd try build this"

Comment (oatsandsugar) [reply README.md:20]: Or dropping some nerd snipes

File: README.md [hunk 4]
```diff
@@ -1,30 +1,17 @@
...
-# Getting started
+[Moose](https://www.getmoose.dev/) is an open-source data engineering framework designed to drastically accellerate AI-enabled software developers, as you prototype and scale data-intensive features and applications.
```
Comment (oatsandsugar) [inline README.md:8]: Not sure what a better word is for that .... but I agree with tim

File: README.md [hunk 5]
```diff
@@ -1,30 +1,17 @@
...
 
-To learn more and/or get started using Moose, check out the documentation: [https://docs.moosejs.com/](https://docs.moosejs.com/)
+To learn more and/or get started using Moose, check out the documentation: [https://docs.getmoose.dev/](https://docs.getmoose.dev/)
```
Comment (oatsandsugar) [inline README.md:10]: I think we should have the getting started guide in the readme.

**Pull request merged**